src/io/flutter/FlutterBundle.properties

```diff
@@ -165,7 +165,7 @@ old.android.studio.message=Your path contains Android Studio 2.x,\n\
 
 flutter.androidstudio.open.module.text=Open Android module in Android Studio
 flutter.androidstudio.open.module.description=Open Android Studio in a new window to edit the top-level Android project
-flutter.androidstudio.open.file.text=Open for Editing in Android Studio
+flutter.androidstudio.open.file.text=Open module in Android Studio
```

Comment:
Since @mit-mit suggested the current label in https://github.com/flutter/flutter-intellij/pull/2216 I think he should sign off for this change.

Reply:
I like that the old string suggested why one would want to open in AS (that the edit part). But perhaps an even longer strong would be better?

Simply 'open module' doesn't really say why one would want to click it.

Reply:
I think it gives a better sense of what will be opened, and generally, think we should aim for brevity here...

Reply:
Sure, but that is only useful if you actually understand what the reason for the dialog is in the first place